Makefile

```diff
@@ -54,16 +54,17 @@ crystal11:       pokecrystal11.gbc
 crystal_au:      pokecrystal_au.gbc
 crystal_debug:   pokecrystal_debug.gbc
 crystal11_debug: pokecrystal11_debug.gbc
+crystalvc:  	 pokecrystalvc.patch
```

Comment:
Mind the alignment. Align with spaces.

Reply:
https://github.com/pret/pokecrystal/commit/42f5de19ea55513a0a940f0614b1b268988b917e should resolve this.